hw/ip/aon_timer/doc/_index.md

```diff
@@ -103,6 +105,8 @@ Clear the interrupt by writing 1 into the Interrupt Status Register {{<regref "I
 
 If the timer has caused a wakeup event ({{<regref"WKUP_CAUSE">}} is set) then clear the wakeup request by writing 0 to {{<regref"WKUP_CAUSE">}}.
 
+If {{<regref"WKUP_COUNT">}} remains above the threshold after clearing the interrupt or the wakeup event and it remains enabled it will trigger again at the next clock tick.
```

Comment:
Again, this is pretty hard to parse and I'm not sure what "it" is for the thing that remains enabled. Maybe this?

```
If {{<regref"WKUP_COUNT">}} remains above the threshold after clearing the interrupt or wakeup event and the timer remains enabled, the interrupt will trigger again at the next clock tick.
```

Reply:
Sounds good, made a slight adjustment as interrupt and wakeup are two seperate things (both of which will get retriggered if you don't do at least one of clear count or disable timer).